make/photon/prepare/templates/nginx/nginx.https.conf.jinja

```diff
@@ -60,7 +60,7 @@ http {
 
     # Recommendations from https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
     ssl_protocols TLSv1.2 TLSv1.3;
-{% if internal_tls.strong_ssl_ciphers %}
+{% if strong_ssl_ciphers %}
```

Comment:
We need to update the harbor.yaml.tmpl file as well.
1, move the strong_ssl_ciphers to the top level.
2, remove the strong_ssl_ciphers under internal_tls.

Reply:
Updated